src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -53,7 +52,7 @@ public void testJumpToLastPage() throws Exception
         sampleSetDataGenerator.insertRows();
 
         CoreComponentsTestPage testPage = CoreComponentsTestPage.beginAt(this, getProjectName());
-        QueryGrid grid = testPage.getQueryGrid("samples", "grid_paging_samples");
+        QueryGrid grid = testPage.getGridPanel("samples", "grid_paging_samples");
```

Comment:
Eventually, once we consolidate our grid use down to GridPanels, we may want to rename QueryGrid (the test wrapper class) to 'GridPanel'.  For now tho, because it has to support both GridPanel and QueryGridPanel, we can leave it as it is

Reply:
Agreed.